docs/source/guide/general/usage.rst

```diff
@@ -260,6 +260,29 @@ key.
 Metadata Update
 ---------------
 
+The Metadata Update ceremony is a guided step-by-step process allowing:
+
+- the removal or addition of key(s)
+- changes to the Root role threshold
+- extending the Root expiration
```

Comment:
Maybe we can use the same verb tense in all structure
```suggestion
- add or remove key(s)
- change the Root role key threshold
- extend the Root expiration
```

Reply:
Maybe it's more precise to say `change the Root expiration` as it could be actually shorten as well.
We override the existing expiration if a user wants to.